test_data/mysql_shard_bulk.cfg

```diff
@@ -0,0 +1,22 @@
+{
+    "configType":"bulk",
+    "shardConfigurationBulk":{
+       "schemaSource":{
+          "host":"test",
+          "user":"test",
+          "password":"test",
+          "port":"3306",
+          "dbName":"test"
+       },
+       "dataShards":[
```

Comment:
Will we not have logical shards in direct connection and have separate dataShard objects for each db although in same physical instance?

Reply:
For direct connection, went with this approach instead its more closer to how bulk migration for a single instance works, but your comments also makes sense. Will come back to this later to see if we need to more closely align the configuration with streaming migration.